### PR TITLE
feat : added category page

### PR DESCRIPTION
**Pull request by raman1999**
image
image

### CODE REVIEW

File: screens/quiz-categories/category.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="./category.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
```
Comment (aishwarya-bm) [inline screens/quiz-categories/category.html:14]: You can add these stylesheets in a common file and import that one in all html files

**Review by aishwarya-bm — COMMENTED** [review]
Overall code looks clean

File: screens/quiz-categories/category.html (hunk 1 above)
Comment (raman1999) [reply screens/quiz-categories/category.html:14]: Thank you for suggestion, I will do it later while clubbing all files together

File: screens/quiz-categories/category.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <li class="list-item  txt-bold">logout</li>
+        </ul>
+    </nav>
+    <h2 class="txt-center txt-theme-3 top-heading">Choose a Category</h2>
+
+    <!-- Movies Categories section -->
+    <p class="category-heading txt-big l-sp-2">MOVIES&nbsp;&nbsp;<i class="fas fa-arrow-right"></i></p>
+    <section class="category-component flex-box">
+        <div class="card">
+            <img src="/images/bolly2.jpg" class="" alt="">
```
Comment (sadathshariff) [inline screens/quiz-categories/category.html]: just a suggestion to give alt attribute some description

**Review by sadathshariff — COMMENTED** [review]
Good use of utility classes and everything looks clean

File: screens/quiz-categories/category.html (hunk 2 above)
Comment (raman1999) [reply screens/quiz-categories/category.html]: Thanks for pointing it out

**Pull request merged**